unumpy/_multimethods.py

```diff
@@ -322,6 +322,17 @@ def accumulate(self, a, axis=0, dtype=None, out=None):
 trunc = ufunc("trunc", 1, 1)
 
 
+@create_numpy(_dtype_argreplacer)
+def empty(shape, dtype=float, order="C"):
```

Comment:
We use strings for dtypes.

```suggestion
def empty(shape, dtype="float64", order="C"):
```